apis/v1alpha1/defaulting.go

```
@@ -98,30 +104,33 @@ func (in *GreptimeDBCluster) SetDefaults() error {
 	if in.Spec.Frontend != nil {
 		defaultGreptimeDBClusterSpec.Frontend = &FrontendSpec{
 			ComponentSpec: ComponentSpec{
-				Replicas: 1,
 				Template: &PodTemplateSpec{},
 			},
 			Service: ServiceSpec{
 				Type: corev1.ServiceTypeClusterIP,
 			},
 		}
+		if in.Spec.Frontend.Replicas == nil || *in.Spec.Frontend.Replicas < 0 {
```

If the replicas < 0, it should throw an error.